File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int_between_1_and_8(string prompt);

int main(void)
{
    int user_input = get_positive_int_between_1_and_8("Height: ");
    for(int i = 1; i <= user_input; i++)
    {
        for(int j = 1; j <= user_input - i; j++)
        {
            printf(" ");
        }
        for(int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for(int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

int get_positive_int_between_1_and_8(string prompt)
{
    int n;
    do
    {
        n = get_int("%s", prompt);
    }
    while(n < 1 || n > 8);
    return n;
}

File: pset6/mario.py
from cs50 import get_int


def main():
    user_input = get_positive_int("Height: ")
    for i in range(1, user_input + 1):
        for j in range(1, (user_input - i) + 1):
            print(" ", end="")

        for k in range(i):
            print("#", end="")

        print("  ", end="")

        for l in range(i):
            print("#", end="")

        print()


def get_positive_int(prompt):
    while True:
        n = get_int(prompt)
        if n >= 1 and n <= 8:
            return n


if __name__ == "__main__":
    main()
